proposals/using-alias-types.md

```diff
@@ -38,20 +38,26 @@ The sections that do need updating are:
 + The namespace_name referenced by a using_namespace_directive is resolved in the same way as the namespace_or_type_name referenced by a using_alias_directive. Thus, using_namespace_directives in the same compilation unit or namespace body do not affect each other and can be written in any order.
 ```
 
+```diff
++ It is illegal for a using alias type to be a nullable reference type.
 
-## Design meeting open questions.
-
-This section needs to be resolved in a design meeting.
+    1. `using X = string?;` is not legal.
+    2. `using X = List<string?>;` is legal.  The alias is to `List<...>` which is itself not a nullable reference type itself, even though it contains one as a type argument.
+    3. `using X = int?;` is legal.  This is a nullable *value* type, not a nullable *reference* type.
+```
 
-The intent of this specification is to allow one to write something like:
+# Supporting aliases to types containing pointers.
 
-```
-using MyPointer = My*;
-```
+A new [unsafe context](https://github.com/dotnet/csharpstandard/blob/standard-v6/standard/unsafe-code.md#222-unsafe-contexts) is added through an optional 'unsafe' modifier in the using_alias_directive production:
 
-The spec is currently unclear if this would be ok or not.  Technically, the `using_alias_directive` here is not in an `unsafe` context, so the `My*` could be considered an error.  However, the spirit of this specification is that should be allowed, and only the *usages* of `MyPointer` would themselves have to either be in another `using_alias_directive` or in an `unsafe` context.  Another way this could be formalized is that the `(namespace_name | type)` portion of a `using_alias_directive` would always be an `unsafe` context, but that wouldn't negate the fact that any place that alias was referenced would also need to be an `unsafe` context.
+```diff
+using_alias_directive
++    : 'using' 'unsafe'? identifier '=' (namespace_name | type) ';'
```

Comment:
Did LDM discuss `unsafe using` versus `using unsafe`? On one hand we have `global using` (keyword before `using`) and on the other we have `using static` (keyword after).

Reply:
We didn't explicitly discuss the two options. We can bring to see how people think about this.

Reply:
I've added it to the next meeting.